### PR TITLE
Add version info for run_fastsurfer and into the docker image

### PR DESCRIPTION
**Pull request by dkuegler**
- Adds extra option to run_fastsurfer.sh: --version with extra option long
  - outputs a version identifier (`{VERSION_TAG}-{git-hash} ({git-branch})`)
  - long option (`--version long`) also adds md5sums of checkpoints and git status to document difference to most recent git commit
- Add VERSION file to docker images
- add version to logfile

### CODE REVIEW

**Comment by m-reuter** [comment]
thanks, but version tags should be read from GitHub (or in the future from the toml file), so that they are only in one place.

**Comment by dkuegler** [comment]
sure, and all but the "VERSION_TAG" is read from git, but right now, we do not support this in our git.

File: run_fastsurfer.sh [hunk 1]
```diff
@@ -247,20 +247,29 @@ EOF
...
+    pushd  "$FASTSURFER_HOME" > /dev/null || return
     VERSION_INFO="$VERSION_TAG-$(git rev-parse --short HEAD) ($(git branch --show-current))"
+    popd > /dev/null || return
+  else
+    VERSION_INFO="$([ -f "$VERSION_FILE" ] && head "$VERSION_FILE" -n 1 || echo "$VERSION_TAG")"
   fi
   if [ "$#" == "1" ] && [ "$1" == "long" ]; then
     pushd "$FASTSURFER_HOME/checkpoints" > /dev/null || return
-    VERSION_INFO=$(printf "%s\ncheckpoints:\n%s" "$VERSION_INFO" "$(md5sum -- *)")
+      VERSION_INFO=$(printf "%s\ncheckpoints:\n%s" "$VERSION_INFO" "$(md5sum -- *)")
```
Comment (m-reuter) [inline run_fastsurfer.sh]: is this indentation on purpose?

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+[build-system]
+requires = ['setuptools >= 61.0.0']
+build-backend = 'setuptools.build_meta'
+
+[project]
+name = 'fastsurfer'
+version = "2.1.1"
```
Comment (m-reuter) [inline pyproject.toml]: Should be 2.2.0-dev.

File: run_fastsurfer.sh (hunk 1 above)
Comment (dkuegler) [reply run_fastsurfer.sh]: yes, to show the pushd / popd context

Comment (dkuegler) [reply run_fastsurfer.sh]: but it was pretty inconsistent :(

**Comment by dkuegler** [comment]
The disadvantage of 2.2.0-dev is that with git hash this looks a bit more ugly: 2.2.0-dev-62df19 vs. 2.2.0dev-62df19
I think that is why without the extra dash is usually the convention in all open source projects that i like, e.g. pytorch with stuff like 1.14a3 or Firefox with 63.1rc1.

File: run_fastsurfer.sh [hunk 3]
```diff
@@ -99,7 +100,8 @@ FLAGS:
                             installed already
   --sid <subjectID>       Subject ID to create directory inside \$SUBJECTS_DIR
   --sd  <subjects_dir>    Output directory \$SUBJECTS_DIR (or pass via env var)
-  --t1  <T1_input>        T1 full head input (not bias corrected). Requires an ABSOLUTE Path!
+  --t1  <T1_input>        T1 full head input (not bias corrected). Requires an
+                            ABSOLUTE Path!
```
Comment (LeHenschel) [inline run_fastsurfer.sh:104]: Why is this on a new line?

Comment (LeHenschel) [reply run_fastsurfer.sh:104]: Looks ok to me. Did you test if this works correctly in docker and native install?

Comment (dkuegler) [reply run_fastsurfer.sh:104]: Newline because of line length

**Comment by dkuegler** [comment]
@LeHenschel finally got the comparison scripts to process everything properly and as I ran both the currently published and this version locally all segmentations and stats files are identical.

It might be that some results differ based on the PC, since @AhmedFaisal95 's sample reference_data has differences to my results.

**Pull request merged**